roborazzi-compose/src/main/java/com/github/takahirom/roborazzi/RoborazziCompose.kt

```diff
@@ -48,20 +42,55 @@ fun captureRoboImage(
   }
 }
 
-/**
- * Workaround for https://github.com/takahirom/roborazzi/issues/100
- */
-private fun registerRoborazziActivityToRobolectricIfNeeded() {
-  try {
-    val appContext: Application = ApplicationProvider.getApplicationContext()
-    Shadows.shadowOf(appContext.packageManager).addActivityIfNotPresent(
-      ComponentName(
-        appContext.packageName,
-        RoborazziTransparentActivity::class.java.name,
-      )
-    )
-  } catch (e: ClassNotFoundException) {
-    // Configured to run even without Robolectric
-    e.printStackTrace()
+fun captureRoboImage(
```

Comment:
I'm not sure, but I think `captureRoboImages` might have the same signature in the JVM bytecode. Could you check it out?

> e: file:///Users/runner/work/roborazzi/roborazzi/sample-android/src/test/java/com/github/takahirom/roborazzi/sample/ManualTest.kt:155:5 Overload resolution ambiguity: